services/s3/src/main/java/software/amazon/awssdk/services/s3/internal/crt/S3CrtAsyncHttpClient.java

```diff
@@ -68,7 +69,9 @@ private S3CrtAsyncHttpClient(Builder builder) {
                                  .withClientBootstrap(s3NativeClientConfiguration.clientBootstrap())
                                  .withPartSize(s3NativeClientConfiguration.partSizeBytes())
                                  .withComputeContentMd5(false)
-                                 .withThroughputTargetGbps(s3NativeClientConfiguration.targetThroughputInGbps());
+                                 .withThroughputTargetGbps(s3NativeClientConfiguration.targetThroughputInGbps())
+                                 .withInitialReadWindowSize(initialWindowSize)
+                                 .withReadBackpressureEnabled(true);
```

Comment:
Query : Do we have a test case to verify BackpressureEnabled is working fine from from java-sdk side ?

Reply:
It's verified through our existing integ tests and stability tests.